README.md

```diff
@@ -6,6 +6,13 @@
 
 Implementation of [Minimum Viable Plasma](https://ethresear.ch/t/minimal-viable-plasma/426)
 
+**Note:** The develop branch contains the current development for v2 of the rootchain contract.
+
+## Project Status
+There is no active development occuring for this project.
```

Comment:
If we change this to "There is very little development", I'm open to merging this in with master.